README.md

```diff
@@ -144,25 +144,22 @@ With this code, on cache miss, the product and its associated images will be loa
 
 ### Caching Polymorphic Associations
 
-IdentityCache tries to figure out both sides of an association whenever it can so it can set those up when rebuilding the object from the cache. In some cases this is hard to determine so you can tell IdentityCache what the association should be. This is most often the case when embedding polymorphic associations. The `inverse_name` option on `cache_has_many` and `cache_has_one` lets you specify the inverse name of the association.
+IdentityCache tries to figure out both sides of an association whenever it can so it can set those up when rebuilding the object from the cache. In some cases this is hard to determine so you can tell IdentityCache what the association should be. This is most often the case when embedding polymorphic associations.
 
 ``` ruby
 class Metafield < ActiveRecord::Base
   include IdentityCache
-  belongs_to :owner, :polymorphic => true
+  belongs_to :owner, polymorphic: true
   cache_belongs_to :owner
 end
 
 class Product < ActiveRecord::Base
   include IdentityCache
-  has_many :metafields, :as => 'owner'
-  cache_has_many :metafields, :inverse_name => :owner
+  has_many :metafields, as: 'owner'
```

Comment:
While we're here, we could change this:
```suggestion
  has_many :metafields, as: :owner
```
Is there test coverage for this? If there is, I think we should assert the error message in it.

Reply:
You mean tests for when the inverse isn't inferred.  Yes, they assert that `IdentityCache::InverseAssociationError` is raised in that case.  I'll add an assertion on the error message.